source/upgrade/v3.txt

```diff
@@ -248,3 +248,7 @@ Version 3.0 Breaking Changes
   - The driver seals the ``MongoClient``, ``MongoDatabase``, and ``MongoCollection`` classes.
     We recommend using the ``IMongoClient``, ``IMongoDatabase``, and ``IMongoCollection``
     interfaces directly.
+
+- The driver requires applications to explicitly configure how to serialize GUIDs
+  by using the ``GuidSerializationProvider`` and ``ObjectSerializer`` classes. To learn
```

Comment:
I think `GuidSerializationProvider` is a typo, there is no such class.

You probably meant `GuidSerializer`.

It might also be worth mentioning that you need to "register" your configured `GuidSerializer` if you want it to apply globally everywhere in your application. `Guid` serialization can also be configured on a case by case basis without registering a `GuidSerializer`.

Registering a global `GuidSerializer` is only safe for NEW applications. For old applications it might be safe but the application writer is then responsible for verifying that all Guids in their application have in fact been serialized the same way in the past.